src/components/modal/createNoteModal/CreateNoteModal.jsx

```diff
@@ -0,0 +1,143 @@
+    tags: [],
+    priority: "",
+  };
+
+  const [noteContent, setNoteContent] = useState(initialNotesData);
+
+  const expandColorPallete = () => {
+    setIsOpen((isOpen) => !isOpen);
+  };
+
+  return (
+    <>
+      <div className="modal-background">
+        <div className="modal-container">
+          <header className="modal-header">
+            <h3>Create Note</h3>
+            <button
+              className="close-icon"
+              onClick={() => setCreateNoteModalVisible(false)}
+            >
+              <IoClose />
+            </button>
+          </header>
+          <form
+            className="add-note-container"
+            style={{ backgroundColor: noteContent.color }}
+          >
+            <input
+              type="text"
+              className="note-title"
+              placeholder="Enter title.."
+              name="title"
+              value={noteContent.title}
+              onChange={(e) =>
+                setNoteContent((prevContent) => ({
+                  ...prevContent,
+                  title: e.target.value,
+                }))
+              }
+            />
+            <textarea
+              id="note-desc"
+              cols="10"
+              rows="10"
+              className="note-content"
+              placeholder="Enter content.."
+              name="content"
+              value={noteContent.content}
+              onChange={(e) =>
+                setNoteContent((prevContent) => ({
+                  ...prevContent,
+                  content: e.target.value,
+                }))
+              }
+            />
+            <div className="add-note-footer">
+              <div className="note-select">
+                <label htmlFor="note-label" className="label md-text">
+                  Label:
+                </label>
+                <select
+                  id="note-label"
+                  className="select"
+                  name="label"
+                  value={noteContent.value}
+                  onChange={(e) =>
+                    setNoteContent((prevContent) => ({
+                      ...prevContent,
+                      tags: [e.target.value],
+                    }))
+                  }
+                >
+                  <option value="">--Choose--</option>
+                  <option value="home">Home</option>
+                  <option value="work">Work</option>
+                </select>
+                <label htmlFor="note-priority" className="label md-text">
+                  Priority:
+                </label>
+                <select
+                  id="note-priority"
+                  className="select"
+                  name="priority"
+                  value={noteContent.priority}
+                  onChange={(e) =>
+                    setNoteContent((prevContent) => ({
+                      ...prevContent,
+                      priority: e.target.value,
+                    }))
+                  }
+                >
```

Comment:
you can directly create on function for change handler and save name and value in object like function chnageHandler(e){ const {name,value}=e.target; setNoteContent ((prevContent) => ({
                      ...prevContent,[name]:value})) like this